src/scripts.js

```diff
@@ -173,16 +173,18 @@ function getRandomUser() {
 }
 
 function showSavedRecipes() {
-  savedView = true;
-  const savedRecipes = currentUser.recipesToCook.map(userRecipe => {
-    return recipeRepo.recipes.find(recipe => recipe.id === userRecipe)
-  });
-  currentView = savedRecipes;
-  searchBar.placeholder = 'Search My Recipes...';
-  filterHeader.innerText = 'Filter My Recipes';
-  toggleHidden(myRecipesTitle);
-  toggleHidden(titleLogo);
-  createRecipeCards(currentView);
+  if(!savedView){
```

Comment:
Live, Laugh, Love this Fix!

Reply:
MIND BLOWING SOLUTION. DOCTORS HATE IT. CHECK OUT THIS ONE EASY CONDITIONAL TO MAKE YOUR CODE LAST FOR HOURS